tf-modules/ingest/variables.tf

```diff
@@ -80,6 +88,14 @@ variable "launchpad_passphrase" {
   default = ""
 }
 
+variable "oauth_provider" {
+  type = string
+}
+
+variable "oauth_user_group" {
```

Comment:
I really dislike having to pass these variables through to this module just so we can make programmatic API requests from a task that is internal to our VPC, but that's a discussion for another day

Reply:
Indeed.     This approach is going to more tightly couple our modules (and indeed suggests and organization issue therein) and make it harder to break up the monolith.   

At a minimum I'd expect we can use config stores instead of variable passing, but I agree with the commentary.